src/main/proto/DataPoint.v2.proto

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Inscope Metrics, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+package com.arpnetworking.kairosdb.proto.v2;
+option java_package = "com.arpnetworking.kairosdb.proto.v2";
+
+message DataPoint {
+    map<uint64, uint32> histogram = 1;
+    double min = 2;
+    double max = 3;
+    double sum = 4;
+    double mean = 5;
+    uint32 precision = 6;
```

Comment:
What's the meaning of precision? Does it really need to be `uint32`?

Reply:
uint32 is the protobuf type that uses variable length encoding.  There are no smaller uint types in protobuf afaik.